packages/hardhat/contracts/Lottery.sol

```diff
@@ -93,10 +93,15 @@ contract Lottery is Ownable {
     /// @notice Calls the bet function `times` times
     function betMany(uint256 times) external {
         require(times > 0);
+        uint256 totalFees = betFee * times;
+        uint256 totalBetPrice = betPrice * times;
+        address[times] memory slotsToPush;
         while (times > 0) {
-            bet();
+            slotsToPush[times - 1] = msg.sender;
```

Comment:
What array is it?

Reply:
the idea was to prepare the array or addresses to push them to slots all at once. But it doesn't seem to be possible, the line 103 is invalid :( 
`_slots.push(slotsToPush);`

Reply:
Got it, I think we may init slotsToPush based on `_slots`
And let's name it's as `slots` maybe?